src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/HttpHeadersUtil.java

```diff
@@ -86,4 +99,55 @@ public static String getHeaderOrDefault(
         final String result = getHeader(responseHeaders, headerName);
         return result == null ? defaulted : result;
     }
+
+    public static boolean hasStatusCode(
+        ListFW<HttpHeaderFW> responseHeaders,
+        int statusCode)
+    {
+        return  (responseHeaders.anyMatch(h ->
+                STATUS.equals(h.name().asString()) && ("" + statusCode).equals(h.value().asString())));
```

Comment:
Conversion of `statusCode` to `String` can be done via `Integer.toString(statusCode)` rather than `"" + statusCode`.